prestoadmin/package.py

```diff
@@ -92,12 +92,20 @@ def rpm_install(rpm_name):
     nodeps = ''
     if env.nodeps:
         nodeps = '--nodeps '
-
-    ret = sudo('rpm -i %s%s' %
-               (nodeps, os.path.join(constants.REMOTE_PACKAGES_PATH,
+   
+    if env.JAVA8_HOME is None:
```

Comment:
Even though on the command line java8_home is all caps, the variable in the code and config file should be lower case to match other env variables.  We use all caps just for constants, and this is something that gets set.